### ISSUE TITLE
Polyfill script tag should be after meta tag

### ISSUE THREAD

**Issue by TatuLund** [vaadin/board#138]
Polyfill script tag should be after meta tag. Otherwise <meta http-equiv="X-UA-Compatible" content=".."> is ignored in IE11. If IE11 is set in compatibility mode, it will not drop from compatibility mode and application will not work.

The problematic code seems to be this: https://github.com/vaadin/webcomponents-helper/blob/master/src/main/java/com/vaadin/webcomponentshelper/polyfill/LoadPolyfill.java#L50

**Comment by alvarezguille** [comment]
Issue has been fixed in vaadin/webcomponents-helper#3 and included in version `0.2.1`
Such version will be included in board once #139 is reviewed and merged.

In the meantime could you verify that you're not able to reproduce the issue anymore, you can do so by temporarily overriding the version for webcomponents-helper to `0.2.1` in the project where you use board:
```
		<dependency>
			<groupId>com.vaadin</groupId>
			<artifactId>webcomponents-helper</artifactId>
			<version>0.2.1</version>
		</dependency>
```

**Comment by alvarezguille** [comment]
Fix was included in 1.0.2 release https://github.com/vaadin/board/releases/tag/1.0.2
And will be included in future 1.1 releases
